### ISSUE TITLE
VAMC Facility template: update Mental health # to use telephone component

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#17745]
## Description
There is a [facility with a Mental health number with an ext](https://www.va.gov/eastern-oklahoma-health-care/locations/yale-avenue-va-clinic/). because this is not using the DS telephone component, the extension is not being parsed correctly and therefore this cannot easily be called directly from mobile phones by just tapping on it. 

This was reported by the Help desk in [this slack thread](https://dsva.slack.com/archives/CDHBKAL9W/p1711648596252219). 

## Additional Context 
This is probably not the only template where this is an issue, it just happens to be the one it's showing on now.

## Steps for Implementation
Update VAMC Facility template to use telephone DS component. 

## Acceptance Criteria
- [ ]  Telephone component has been added, including the proper variant when necessary
- [ ] Requires accessibility review

**Comment by laflannery** [comment]
@jilladams  is this a duplicate of #17745 or just a more targeted ticket for priority purposes?

**Comment by jilladams** [comment]
I think this was me in a state of confusion about the preponderance of telephone tickets. Closing asdupe.
